master/buildbot/scripts/base.py

```diff
@@ -113,20 +113,25 @@ def print_error(error_message):
     return True
 
 
-def getConfigFromTac(basedir):
+def getConfigFromTac(basedir, quiet=False):
     tacFile = os.path.join(basedir, 'buildbot.tac')
     if os.path.exists(tacFile):
         # don't mess with the global namespace, but set __file__ for
         # relocatable buildmasters
         tacGlobals = {'__file__': tacFile}
-        exec(open(tacFile).read(), tacGlobals)
+        try:
+            exec(open(tacFile).read(), tacGlobals)
+        except Exception as e:
+            if not quiet:
+                print(e)
```

Comment:
The exception is printed in the checkconfig method and now also here. Could this result in an exception being printed twice?